src/display/text_layer.js

```diff
@@ -213,19 +531,85 @@ var renderTextLayer = (function renderTextLayerClosure() {
       var styles = this._textContent.styles;
       var textDivs = this._textDivs;
       var viewport = this._viewport;
+      var bounds = [];
+      var enhanceTextSelection = this._enhanceTextSelection;
+      var enhanceOnClick = this._enhanceOnClick;
       for (var i = 0, len = textItems.length; i < len; i++) {
-        appendText(textDivs, viewport, textItems[i], styles);
+        appendText(textDivs, viewport, textItems[i], styles, bounds,
+                   enhanceTextSelection);
+      }
+      if (enhanceTextSelection) {
+        expand(bounds, this._viewport);
+        if (enhanceOnClick) {
+          this._bindMouse(textDivs);
+        }
       }
-
       if (!timeout) { // Render right away
-        render(this);
+        render(this, enhanceOnClick);
```

Comment:
Comment https://github.com/mozilla/pdf.js/pull/7539/commits/a02d66a1089586a29f1afb7e2338257aa0e73374#r75456925 applies both here, and below as well.